### PR TITLE
adds a couple of presentations

### PR DESCRIPTION
**Pull request by Frijol**
Helps our site have post-2017 content.

Not sure if this is the best formatting/intended linking for this section?

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -156,6 +156,16 @@ <h2 class="tagline">Publications and Talks</h2>
...
         <p>Slides: <a href="./presentations/Data%20Together%20-%20NDSR%20-%20swadeshi.pdf">PDF</a>, <a href="./presentations/Data%20Together%20-%20NDSR%20-%20swadeshi.key">Keynote</a> </p>
       </div>
+      <div class="text col-md-4">
+        <p><a href="https://youtu.be/83Gws-dPL8A">The Future of the Web – CascadiaJS 2018</a></p>
+        <iframe src="https://www.youtube.com/embed/83Gws-dPL8A" max-height="315" width="100%" frameborder="0" webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen></iframe>
+        </p>
+      </div>
+      <div class="text col-md-4">
+        <p><a href="https://youtu.be/GcPU1Os2-Q8">Where is the data bazaar?</a></p>
+        <iframe src="https://www.youtube.com/embed/GcPU1Os2-Q8" max-height="315" width="100%" frameborder="0" webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen></iframe>
```
Comment (dcwalk) [inline docs/index.html:166]: I'm getting an error that this video link is unavailable

File: docs/index.html [hunk 2]
```diff
@@ -156,6 +156,16 @@ <h2 class="tagline">Publications and Talks</h2>
         <p>Swadeshi (starts at 1:10)</p>
         <p>Slides: <a href="./presentations/Data%20Together%20-%20NDSR%20-%20swadeshi.pdf">PDF</a>, <a href="./presentations/Data%20Together%20-%20NDSR%20-%20swadeshi.key">Keynote</a> </p>
       </div>
+      <div class="text col-md-4">
+        <p><a href="https://youtu.be/83Gws-dPL8A">The Future of the Web – CascadiaJS 2018</a></p>
```
Comment (dcwalk) [inline docs/index.html:160]: This done doesn't appear to mention Data Together (skim-viewed it), does it fit? what is the role of these videos? 

I wonder if something from @b5 , maybe at Our Networks 2018: https://youtu.be/4mIeV890ymg

**Review by dcwalk — COMMENTED** [review]
Left some thoughts inline

File: docs/index.html (hunk 2 above)
Comment (Frijol) [reply docs/index.html:160]: Oh, I think I had a different idea in mind re what these videos are (does Zumwalt's video mention Data Together?)– this is a presentation that's on-theme from one of the DT partners. Do we only link explicitly Data Together content?

File: docs/index.html (hunk 1 above)
Comment (Frijol) [reply docs/index.html:166]: strange, it's working ok for me

**Pull request closed without merging**